README.md

```diff
@@ -57,6 +57,7 @@ Install them using Homebrew:
 
 ```sh
 brew install cmake pkg-config
+brew install graphviz
```

Comment:
Out of interest, do you remember what needed `graphviz`? 🤔 I have no idea what it's being used for.

Also, I'm wondering if it could just be tacked on the end of this line?

```suggestion
brew install cmake pkg-config graphviz
```

Reply:
Oh yeah, that's a nicer solution!

I noted that I needed to install it and totally forgot what asked for it 🙄 a cursory bit of research suggests that it'd a dependency of `rails-erd`.